city_scrapers/spiders/pitt_allegheny_league.py

```diff
@@ -0,0 +1,133 @@
+        calendar_data = unicodedata.normalize("NFKD", calendar_data)
+        # Split events
+        calendar_data = calendar_data.split('\n See more details')
+        calendar_href = response.xpath('//div[@class="simcal-event-details"]//a/@href').extract()
+
+        for index, item in enumerate(calendar_data):
+            item = os.linesep.join([s.replace('\t', '').strip()
+                                   for s in item.splitlines()
+                                   if s.rstrip()]).split('\n')
+
+            if len(item) < 3:
+                continue
+            data = {
+                '_type': 'event',
+                'name': item[1],
+                'event_description': item[1],
+                'classification': self._parse_classification(item),
+                'start': self._parse_start(item),
+                'end': self._parse_end(item),
+                'all_day': self._parse_all_day(item),
+                'location': self._parse_location(item),
+                'documents': self._parse_documents(calendar_href[index]),
+                'sources': self._parse_sources(calendar_href[index]),
+            }
+
+            data['status'] = self._generate_status(data,
+                                                   item[-1])
+            data['id'] = self._generate_id(data)
+
+            yield data
+
+        # self._parse_next(response) yields more responses to parse if necessary.
+        # uncomment to find a "next" url
+        # yield self._parse_next(response)
+
+    def _parse_start(self, item):
+        """
+        Parse start date and time.
+        """
+        start, _ = item[2].split('-')
+        datetime_obj = datetime.strptime(start.strip(),
+                                         '%B %d, %Y @ %I:%M %p')
+        return {
+            'date': datetime_obj.date(),
+            'time': datetime_obj.time(),
+            'note': ''
+        }
+
+    def _parse_end(self, item):
+        """
+        Parse end date and time.
+        """
+        _, end = item[2].split('-')
+        datetime_obj = datetime.strptime(('{date} {time}'
+                                          .format(date=item[0], time=end)),
+                                         '%B %d, %Y %I:%M %p')
+        return {
+            'date': datetime_obj.date(),
+            'time': datetime_obj.time(),
+            'note': ''
+        }
+
+    def _parse_all_day(self, item):
+        """
+        Parse or generate all-day status. Defaults to False.
+        """
+        return False
+
+    def _parse_location(self, item):
+        """
+        Parse or generate location. Latitude and longitude can be
+        left blank and will be geocoded later.
+        """
+        return {
+            'address': item[-1],
+            'name': '',
+            'neighborhood': '',
+        }
+
+    def _parse_documents(self, item):
+        """
+        Parse or generate documents.
+        """
+        return [{'url': item, 'note': ''}]
```

Comment:
Since there can be multiple documents with different information, we should always have a note for documents. In this case, it looks like it's just giving a link to Google Calendar, so there aren't any documents available and this can be dropped. `sources` is a bit confusing since it's really just the link the page was scraped from which is why we're allowing there to be an empty note